frontend/components/ValueViewer.vue

```diff
@@ -50,13 +67,38 @@ export default {
   },
 
   async mounted () {
-    this.valueToView = await this.$wikibase.getWbValue(this.value.property, this.value.datatype, this.value.datavalue.value, this.$i18n.locale)
+    this.valueToView = await this.$wikibase.getWbValue(
+      this.value.property,
+      this.value.datatype,
+      this.value.datavalue.value,
+      this.$i18n.locale
+    )
     this.datavalue = this.valueToView.value
+
+    if (this.isURL(this.valueToView.value) === true) {
+      console.log(this.valueToView.value)
+      // await this.$image.getTopImageFromUrl(this.valueToView.value)
+
+      const res = await fetch(
+        'https://the-visionary-git-master-austininseoul.vercel.app/api?url=' +
```

Comment:
Interesting! Let's discuss it, maybe we can include this service in our application.

Reply:
is there a way to include it without adding an express server to the frontend? since these request must be done server side. ie. to flickr.com

Reply:
no but we already have a backend maybe we could implement it there
anyway we can discuss it later in our meetings, for now is ok!